src/components/entity/state-badge.ts

```diff
@@ -205,6 +224,21 @@ export class StateBadge extends LitElement {
           box-sizing: border-box;
           --state-inactive-color: initial;
         }
+        :host(.has-image) {
+          border-radius: var(
+            --state-badge-with-image-border-radius,
+            50%
+          ) !important;
+        }
+        :host(.has-media-image) {
+          border-radius: var(
+            --state-badge-with-media-image-border-radius,
+            8%
+          ) !important;
+        }
+        :host(.has-no-radius) {
+          border-radius: 0 !important;
+        }
```

Comment:
I don't think we need `!important` on these since the selectors are already more specific than `:host`

Reply:
done